src/main/java/com/linkedin/kmf/services/ProduceService.java

```diff
@@ -77,14 +79,17 @@
   private final int _threadsNum;
   private final String _zkConnect;
   private final boolean _treatZeroThroughputAsUnavailable;
+  private final int _latencyPercentileMaxMs;
+  private final int _latencyPercentileGranularityMs;
 
   public ProduceService(Map<String, Object> props, String name) throws Exception {
     _name = name;
     ProduceServiceConfig config = new ProduceServiceConfig(props);
     _zkConnect = config.getString(ProduceServiceConfig.ZOOKEEPER_CONNECT_CONFIG);
     _brokerList = config.getString(ProduceServiceConfig.BOOTSTRAP_SERVERS_CONFIG);
     String producerClass = config.getString(ProduceServiceConfig.PRODUCER_CLASS_CONFIG);
-
+    _latencyPercentileMaxMs = config.getInt(ProduceServiceConfig.LATENCY_PERCENTILE_MAX_MS_CONFIG);
```

Comment:
It's not clear what this is supposed to be.  By reading the code it looks like percentiles are based on some fixed sized histogram and this is the maximum value allowed by the histogram.   Perhaps maxLatencyHistogram or maxBinnedValue or something like that would be better.

Reply:
Thanks for the review!

Yes the implementation of this metric is based on bucket. On the other hand the metric exposed to user is actually a percentile number rather than histogram. So it maybe better to have `percentile` in the metric name to refer to the interpretation of the metric rather than using `histogram` to refer to the implementation of this metric.

Also, this config is very similar to the existing config `consume.latency.percentile.max.ms` in the ConsumeServiceConfig. So it maybe better to follow the same pattern for consistency?